alexandra_sirbu/main.cpp

```diff
@@ -0,0 +1,47 @@
+#include <fstream>
+#include <iostream>
+#include "graph.hpp"
+
+constexpr int kVerticesCount = 14;
+
+void writeGraphJSON(const Graph& graph, const std::string& filename) {
+  std::ofstream json(filename);
+  if (json.is_open()) {
+    json << graph.toJSON();
+    json.close();
+    std::cout << "Done! :)\n";
+  } else {
+    std::runtime_error("Unable to open file :(");
```

Comment:
Ты просто создала объект ошибки, в чем нет смысла, нужно выкинуть эту ошибку
```suggestion
    throw std::runtime_error("Unable to open file :(");
```